src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -197,26 +197,39 @@ function presentTransaction (transaction) {
 const HoloFuelDnaInterface = {
   user: {
     get: async () => {
-      const result = await createZomeCall('transactions/whoami')()
-      if (result.Err) throw new Error('There was an error locating the current holofuel agent nickname. ERROR: ', result.Err)
-
+      const myProfile = await createZomeCall('profile/get_my_profile')()
+      if (myProfile.Err) throw new Error('There was an error locating the current holofuel agent profile. ERROR: ', myProfile.Err)
```

Comment:
Error doesn't concatenate strings, so you need to use a template literal
`There was an error locating the current holofuel agent profile. ERROR: ${myProfile.Err}`